<ui version="4.0" >
 <class>a2bInstaller</class>
 <widget class="QMainWindow" name="a2bInstaller" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>238</height>
   </rect>
  </property>
  <property name="cursor" >
   <cursor>0</cursor>
  </property>
  <property name="windowTitle" >
   <string>a2B installer</string>
  </property>
  <property name="windowIcon" >
   <iconset/>
  </property>
  <widget class="QWidget" name="centralWidget" >
   <layout class="QGridLayout" >
    <property name="margin" >
     <number>9</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item row="0" column="0" colspan="5" >
     <widget class="QWidget" native="1" name="mainWidget" >
      <property name="sizePolicy" >
       <sizepolicy>
        <hsizetype>7</hsizetype>
        <vsizetype>7</vsizetype>
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" colspan="3" >
        <widget class="QTextBrowser" name="textField" >
         <property name="frameShape" >
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Plain</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QLineEdit" name="lineEdit" />
       </item>
       <item row="1" column="2" >
        <widget class="QPushButton" name="detailButton" >
         <property name="text" >
          <string>参照...(&amp;F)</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" colspan="5" >
     <widget class="Line" name="line" >
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="4" >
     <widget class="QPushButton" name="btnAbort" >
      <property name="text" >
       <string>中止 (&amp;A)</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3" >
     <spacer>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType" >
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="2" >
     <widget class="QPushButton" name="btnNext" >
      <property name="text" >
       <string>次へ (&amp;N) ></string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" >
     <widget class="QPushButton" name="btnBack" >
      <property name="text" >
       <string>&lt; 戻る (&amp;B)</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" >
     <spacer>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>131</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <action name="actionClose" >
   <property name="icon" >
    <iconset>batsu.png</iconset>
   </property>
   <property name="text" >
    <string>Close (&amp;C)</string>
   </property>
   <property name="iconText" >
    <string>Close (&amp;C)</string>
   </property>
   <property name="toolTip" >
    <string>Close (&amp;C)</string>
   </property>
   <property name="shortcut" >
    <string>Q</string>
   </property>
  </action>
  <action name="actionAbout_Qt" >
   <property name="icon" >
    <iconset>Qt-icon.png</iconset>
   </property>
   <property name="text" >
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionCapture" >
   <property name="icon" >
    <iconset>camera-icon.png</iconset>
   </property>
   <property name="text" >
    <string>Capture</string>
   </property>
   <property name="shortcut" >
    <string/>
   </property>
  </action>
  <action name="actionCaptureScreen" >
   <property name="text" >
    <string>Capture Screen!</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>actionClose</sender>
   <signal>triggered()</signal>
   <receiver>a2bInstaller</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>135</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
